### PR TITLE
Mark unsupported projects as such

### PR DESCRIPTION
**Pull request by ludwigbald**
- I marked 84 projects that have seen no new activity since 2019-01-01 as "Not actively maintained", during a boring lecture (#744)

### CODE REVIEW

File: gatsby/content/projects/2015-11-28-matrixtool.mdx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 layout: project
 title: MatrixTool
-categories: 
+categories:
  - other
 description: Commands to interact with a Matrix homeserver
 author: LeoNerd
 language: Perl
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2015-11-28-matrixtool.mdx:9]: Perl tends to have longer-term thinking, can we revert this file please

File: gatsby/content/projects/2015-12-10-glib-sdk.mdx [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: projectimage
 title: Matrix Client SDK for GLib
-categories: 
+categories:
  - sdk
 description: Matrix Client SDK for GLib
 author: Gergely Polonkai
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2015-12-10-glib-sdk.mdx:8]: revert this one

File: gatsby/content/projects/2015-12-04-hubot-matrix.mdx [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 layout: project
 title: Hubot-Matrix
-categories: 
+categories:
  - bot
 description: A Matrix-adapter for Hubot
 author: David A Roberts
 language: JavaScript
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2015-12-04-hubot-matrix.mdx:9]: this has commits in the last 9 months and is in use, please revert

File: gatsby/content/projects/2016-01-30-rocket-chat-federation.mdx [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 title: Rocket Chat Federation
-categories: 
+categories:
  - as
 thumbnail: https://raw.githubusercontent.com/Sing-Li/bbug/master/images/rcsnynapse.png
 author: Rocket.Chat
 language: JavaScript
 license: MIT
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-01-30-rocket-chat-federation.mdx:10]: would rather revert this please

File: gatsby/content/projects/2016-02-03-feedbot.mdx [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
 title: feedbot
-categories: 
+categories:
  - bot
  - bridge
 thumbnail: /docs/projects/images/rss.svg
 description: Connects to RSS and Twitter feeds
 author: Ryan Rix
-maturity: Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-02-03-feedbot.mdx:10]: this is still in use, please revert

File: gatsby/content/projects/2016-02-03-lightrix.mdx [hunk 6]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: projectimage
 title: Lightrix
-categories: 
+categories:
  - other
 description: Drive Adafruit Neopixels over Matrix
 author: Ryan Rix
-maturity: Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-02-03-lightrix.mdx:8]: no recent commits != abandoned, please revert

File: gatsby/content/projects/2016-02-03-polynomial.mdx [hunk 7]
```diff
@@ -1,18 +1,16 @@
 ---
 layout: projectimage
 title: Polynomial
-categories: 
+categories:
  - other
 description: A Decentralized Webring
 author: Ryan Rix
-maturity: Early Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-02-03-polynomial.mdx:8]: pls revert

File: gatsby/content/projects/2016-02-05-net-async-matrix-perl.mdx [hunk 8]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: projectimage
 title: Net::Async::Matrix (Perl)
-categories: 
+categories:
  - sdk
 description: 
 author: LeoNerd
-maturity: Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-02-05-net-async-matrix-perl.mdx:8]: please revert

File: gatsby/content/projects/2016-03-19-mpd-dj.mdx [hunk 9]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: projectimage
 title: MPD DJ
-categories: 
+categories:
  - bot
 description: A bot for controlling MPD over matrix.
 author: Half-Shot
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-03-19-mpd-dj.mdx:8]: please revert

File: gatsby/content/projects/2016-04-30-concourse-matrix-notification-resource.mdx [hunk 10]
```diff
@@ -1,18 +1,16 @@
...
 layout: project
 title: Concourse/Matrix notification resource
-categories: 
+categories:
  - other
 thumbnail: /docs/projects/images/concourse-ci-logo.png
 description: Post notifications from Concourse CI jobs
 author: freelock
-maturity: Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-04-30-concourse-matrix-notification-resource.mdx:9]: please revert, we may revisit

File: gatsby/content/projects/2016-07-29-matrix-appservice-rocketchat.mdx [hunk 11]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: matrix-appservice-rocketchat
-categories: 
+categories:
  - as
 description: This project bridges Rocket Chat to Matrix.
 author: oddvar
-maturity: Early Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-07-29-matrix-appservice-rocketchat.mdx:8]: please revert, it's just (long) paused

File: gatsby/content/projects/2016-08-11-nervewire.mdx [hunk 12]
```diff
@@ -1,14 +1,14 @@
...
-categories: 
+categories:
  - other
 description: A Matrix.org image deck
 author: Ryan Rix
-maturity: Early Beta
-language: 
-license: 
-repo: 
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-08-11-nervewire.mdx:8]: please revert

File: gatsby/content/projects/2016-09-04-hello-matrix-bot.mdx [hunk 13]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: Hello Matrix Bot
-categories: 
+categories:
  - bot
 description: Bot with an array of plugins
 author: Alexander Rudyk
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-09-04-hello-matrix-bot.mdx:8]: please revert

File: gatsby/content/projects/2016-09-05-node-red-nodes.mdx [hunk 14]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: Matrix bot nodes for Node-RED
-categories: 
+categories:
  - other
 description: Visual bot creation using Node-RED
 author: mlopezr
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-09-05-node-red-nodes.mdx:8]: please revert

File: gatsby/content/projects/2016-09-17-drupal-matrix-api.mdx [hunk 15]
```diff
@@ -1,14 +1,14 @@
...
 layout: projectimage
 title: Drupal matrix_api module
-categories: 
+categories:
  - sdk
 thumbnail: /docs/projects/images/200px-druplicon.png
 description: A Drupal 8 integration module/SDK
 author: freelock
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-09-17-drupal-matrix-api.mdx:9]: please revert

File: gatsby/content/projects/2016-10-04-matrix-dotnet-framework.mdx [hunk 16]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: matrix-dotnet-framework
-categories: 
+categories:
  - other
 description: Bot framework for matrix-dotnet-sdk
 author: Half-Shot
-maturity: Early Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-10-04-matrix-dotnet-framework.mdx:8]: please revert

File: gatsby/content/projects/2016-10-16-matrix-esp8266.mdx [hunk 17]
```diff
@@ -1,18 +1,16 @@
 ---
 layout: project
 title: matrix-esp8266
-categories: 
+categories:
  - other
 description: Tiny matrix client for the esp8266 microcontroller
 author: Matt Williams
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-10-16-matrix-esp8266.mdx:8]: please revert

File: gatsby/content/projects/2016-10-18-riotic.mdx [hunk 18]
```diff
@@ -1,16 +1,14 @@
 ---
 layout: project
 title: Riotic
-categories: 
+categories:
  - client
 description: macOS native webkitwrapper of Riot
 author: Joakim Ahlen
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-10-18-riotic.mdx:8]: please revert

File: gatsby/content/projects/2016-12-11-riotchat-ansible.mdx [hunk 19]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: riotchat-ansible
-categories: 
+categories:
  - other
 description: Ansible playbook for matrix/riot.im setup
 author: kultsinuppeli
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-12-11-riotchat-ansible.mdx:8]: please revert

File: gatsby/content/projects/2016-12-12-matrix-fb-chat.mdx [hunk 20]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: matrix-fb-chat
-categories: 
+categories:
  - other
 description: POC designed to bridge Matrix rooms to FB chat.
 author: Half-Shot
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-12-12-matrix-fb-chat.mdx:8]: please revert

File: gatsby/content/projects/2016-12-26-matrix-music-bot.mdx [hunk 21]
```diff
@@ -1,18 +1,16 @@
 ---
 layout: project
 title: matrix-music-bot
-categories: 
+categories:
  - bot
 description: last.fm scrobbler
 author: Falko
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2016-12-26-matrix-music-bot.mdx:8]: please revert

File: gatsby/content/projects/2017-01-05-synpurge.mdx [hunk 22]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: synpurge
-categories: 
+categories:
  - other
 description: Purges Matrix room history room using the HTTP API
 author: Adrian Perez de Castro
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2017-01-05-synpurge.mdx:8]: please revert

File: gatsby/content/projects/2017-01-31-matrix-recorder.mdx [hunk 23]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: Matrix Recorder
-categories: 
+categories:
  - other
 description: Enables you to keep a record of all messages you have received.
 author: Alexander Rudyk
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2017-01-31-matrix-recorder.mdx:8]: please revert

File: gatsby/content/projects/2017-02-05-navi.mdx [hunk 24]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: Navi
-categories: 
+categories:
  - bot
 description: A minimal Matrix notification bot
 author: Maximilian Fricke
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2017-02-05-navi.mdx:8]: please revert

File: gatsby/content/projects/2017-02-06-smartparking.mdx [hunk 25]
```diff
@@ -1,18 +1,16 @@
 ---
 layout: project
 title: SmartParking
-categories: 
+categories:
  - other
 description: A parking system PoC using Matrix
 author: Natan Bagrov, Ran Moshe
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2017-02-06-smartparking.mdx:8]: please revert

File: gatsby/content/projects/2017-02-23-matrix-appservice-instagram-media.mdx [hunk 26]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: matrix-appservice-instagram-media
-categories: 
+categories:
  - bridge
 description: Bridges Instagram to Matrix
 author: Travis Ralston
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2017-02-23-matrix-appservice-instagram-media.mdx:8]: please revert

File: gatsby/content/projects/2017-08-18-matrix-appservice-twilio.mdx [hunk 27]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: matrix-appservice-twilio
-categories: 
+categories:
  - as
 description: Bridges Twilio (SMS) to Matrix
 author: Travis Ralston
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2017-08-18-matrix-appservice-twilio.mdx:8]: please revert

File: gatsby/content/projects/2017-09-13-godot-matrix.mdx [hunk 28]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: projectimage
 title: godot-matrix
-categories: 
+categories:
  - sdk
 description: Godot Engine module for Matrix client
 author: vurpo
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2017-09-13-godot-matrix.mdx:8]: please revert

File: gatsby/content/projects/2018-03-08-matrix-mbot.mdx [hunk 29]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: matrix-mbot
-categories: 
+categories:
  - bot
 description: Extensible bot based on Net::Async::Matrix
 author: Marko Punnar
-maturity: Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-03-08-matrix-mbot.mdx:8]: please revert, perl is longer term

File: gatsby/content/projects/2018-04-24-Operator.mdx [hunk 30]
```diff
@@ -1,16 +1,14 @@
 ---
 layout: projectimage
 title: Operator
-categories: 
+categories:
  - sdk
 description: Operator is a C++ library for plugging into the Matrix
 author: uhoreg
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-04-24-Operator.mdx:8]: please revert

File: gatsby/content/projects/2018-04-24-kacki.mdx [hunk 31]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: project
 title: kacki
-categories: 
+categories:
  - client
 description: Simple chatbot for matrix.org, inspired by rbot.
 author: synopia
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-04-24-kacki.mdx:8]: please revert

File: gatsby/content/projects/2018-04-24-matrix-appservice-prosody.mdx [hunk 32]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: matrix-appservice-prosody
-categories: 
+categories:
  - as
 description: Prosody module to act as a Matrix Application Service
 author: uhoreg
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-04-24-matrix-appservice-prosody.mdx:8]: please revert

File: gatsby/content/projects/2018-04-24-matrix-olm-cpp.mdx [hunk 33]
```diff
@@ -1,16 +1,14 @@
 ---
 layout: project
 title: matrix-olm-cpp
-categories: 
+categories:
  - other
 description: Provides a C++ wrapper for the libolm library, used in Matrix's end-to-end encryption
 author: ajberchek
-maturity: Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-04-24-matrix-olm-cpp.mdx:8]: please revert

File: gatsby/content/projects/2018-04-24-matrix-xmpp-bridge.mdx [hunk 34]
```diff
@@ -1,18 +1,18 @@
...
 layout: project
 title: matrix-xmpp-bridge
-categories: 
+categories:
  - as
+ - bridge
 description: This project creates a bridge between a Matrix room and an XMPP MUC
 author: jfrederickson
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-04-24-matrix-xmpp-bridge.mdx:9]: please revert

File: gatsby/content/projects/2018-04-24-uMatriks.mdx [hunk 35]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: uMatriks
-categories: 
+categories:
  - client
 description: uMatriks is a native matrix client being developed for Ubuntu Touch
 author: uMatriks team
-maturity: Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-04-24-uMatriks.mdx:8]: please revert

File: gatsby/content/projects/2018-06-28-python-olm.mdx [hunk 36]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: project
 title: python-olm
-categories: 
+categories:
  - other
 description: Python bindings for the Olm C library.
 author: poljar
-maturity: Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-06-28-python-olm.mdx:8]: please revert

File: gatsby/content/projects/2018-07-16-olm-rs.mdx [hunk 37]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: project
 title: olm-rs
-categories: 
+categories:
  - other
 description: olm, implemented in Rust
 author: musoke
-maturity: Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-07-16-olm-rs.mdx:8]: please revert

File: gatsby/content/projects/2018-07-18-matrix-dialogflow.mdx [hunk 38]
```diff
@@ -1,12 +1,12 @@
...
 layout: projectimage
 title: matrix-dialogflow
-categories: 
+categories:
  - bot
 thumbnail: /docs/projects/images/Dialogflow_logo.svg
 description: Connect a DialogFlow agent to Matrix as a chatbot
 author: vurpo
-maturity: Beta
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-07-18-matrix-dialogflow.mdx:9]: please revert

File: gatsby/content/projects/2018-09-03-racket-matrix-sdk.mdx [hunk 39]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: projectimage
 title: Racket Matrix SDK
-categories: 
+categories:
  - sdk
 description: A Matrix client library written in Racket
 author: Aidan Gauland
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-09-03-racket-matrix-sdk.mdx:8]: please revert

File: gatsby/content/projects/2018-12-10-EmacsMaelstrom.mdx [hunk 40]
```diff
@@ -1,16 +1,14 @@
 ---
 layout: projectimage
 title: Emacs Maelstrom
-categories: 
+categories:
  - client
 description: Library and client for the Matrix protocol for Emacs
 author: skolar42
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-12-10-EmacsMaelstrom.mdx:8]: please revert

File: gatsby/content/projects/2018-12-10-matrix-appservice-hangouts.mdx [hunk 41]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: projectimage
 title: matrix-appservice-hangouts
-categories: 
+categories:
  - bridge
 description: Hangouts appservice written in Python 3.6 using asyncio.
 author: Cadair
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-12-10-matrix-appservice-hangouts.mdx:8]: please revert

File: gatsby/content/projects/2018-12-10-matrix-appservice-hangouts.mdx [hunk 42]
```diff
@@ -1,15 +1,15 @@
...
 language: Python
 license: MIT
 repo: https://github.com/Cadair/matrix-appservice-hangouts
-featured: true
+featured: false
```
Comment (benparsons) [inline gatsby/content/projects/2018-12-10-matrix-appservice-hangouts.mdx:12]: please revert

File: gatsby/content/projects/2018-12-13-mxtoot.mdx [hunk 43]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: projectimage
 title: MXToot
-categories: 
+categories:
  - bridge
 description: A Matrix<->Mastodon bot written in Java.
 author: ma1uta
-maturity: Alpha
+maturity: Not actively maintained
```
Comment (benparsons) [inline gatsby/content/projects/2018-12-13-mxtoot.mdx:8]: please revert

**Comment by SuperSandro2000** [comment]
You wrote on almost all projects revert this. If I project did not receive a commit for two years it is not actively maintained. It may still work fine but it definitely did not receive any security updates in the last two years. Also the metric still in use does not count. I can use 40 year old software but I shouldn't in production or in chat bridges where people can feed to program almost anything they want.

File: gatsby/content/projects/2018-12-10-matrix-appservice-hangouts.mdx (hunk 41 above)
Comment (SuperSandro2000) [reply gatsby/content/projects/2018-12-10-matrix-appservice-hangouts.mdx:8]: Just to give one example. This bridge is currently broken, requires a patch and no one reacted to this [issue](https://github.com/Cadair/matrix-appservice-hangouts/issues/249) in over 14 months. Does not look like it is maintained or should be featured.

**Comment by ludwigbald** [comment]
I tried to leave things stable that were previously "stable".

But think previously "alpha" things that are not being worked on are probably mostly dead projects.

**Comment by benparsons** [comment]
Well, I'm asking to revert around half of the original list, not all of it. I agree that if a project hasn't been worked on for some length of time it probably isn't active, and we can probably mark it as such.

NB the main thing this field does is control what is visible by default on try-matrix-now, which is a page with considerably less traffic than the other landing pages such as /clients, /sdks and /bridges.

For now, I'll merge this as is, and we'll keep an eye on the marked projects

**Comment by ludwigbald** [comment]
Cool! With my other PR #756 the list with all of these items is moved to the Discover page.

**Pull request merged**